src/lokalise/types.ts

```diff
@@ -0,0 +1,115 @@
+import { PaginatedResult } from '@lokalise/node-api';
+
+/**
+ * Based on the locale codes used in the file paths.
+ */
+export enum COLOR_LANGUAGE_ISO_CODE {
```

Comment:
Can we remove the `COLOR` prefix here? This is an open source project so we should not couple our code directly if we can help it.